projects/step-core/src/lib/services/export-dialogs.service.ts

```diff
@@ -1,15 +1,15 @@
-import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs';
-import { UibModalHelperService } from './uib-modal-helper.service';
+import { inject, Injectable } from '@angular/core';
+import { catchError, Observable, of } from 'rxjs';
 import { a1Promise2Observable } from '../shared';
+import { UibModalHelperService } from './uib-modal-helper.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class ExportDialogsService {
-  constructor(private _uibModalHelper: UibModalHelperService) {}
+  private _uibModalHelper = inject(UibModalHelperService);
 
-  displayExportDialog<T>(title: string, path: string, filename: string): Observable<T> {
+  displayExportDialog(title: string, path: string, filename: string): Observable<boolean | void> {
```

Comment:
Such return type looks strange.
If we're only interested it, will the error be returned or not - than it should be boolean only